app/views/layouts/application.html.erb

```diff
@@ -22,6 +26,10 @@
         <%= link_to 'Login', new_user_session_path, class: 'navbar-link'  %>
       <% end %>
     </p>
+    
     <%= yield %>
+    <%= render(partial: 'footer') %>
```

Comment:
These parentheses are not required, for better readability is better to remove them.